new-lamassu-admin/src/pages/ServerLogs.js

```diff
@@ -55,6 +57,40 @@ styles.uptimeContainer = {
 styles.titleAndButtonsContainer = titleAndButtonsContainer
 styles.buttonsWrapper = buttonsWrapper
 
+styles.popoverContent = {
+  minWidth: 315
+}
+
+styles.popoverHeader = {
+  extend: h4,
+  padding: [[20, 15, 0, 15]]
+}
+
+styles.radioButtonsContainer = {
+  padding: [[10, 15, 10, 15]]
+}
+
+styles.radioButtons = {
+  display: 'flex',
+  justifyContent: 'space-between',
+  flexDirection: 'row',
+  color: primaryColor
+}
+
+styles.dateRangePickerShowing = {
+  display: 'block',
+  height: '100%'
+}
+
+styles.dateRangePickerHidden = {
+  display: 'none',
+  height: 0
+}
+
+styles.download = {
+  padding: [[30, 15, 30, 15]]
+}
+
```

Comment:
Why aren't these on the style files?

Reply:
Resolved in https://github.com/lamassu/lamassu-server/pull/328/commits/1dbb47aa94c828e4033ab65b63bb6fa3f4a970fc.